src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStringParts = sourceString.split(';');
+  const strWithoutSpaces = sourceStringParts.map((str) => str.trim());
+  const result = strWithoutSpaces.reduce((prev, el) => {
+    const [key, value] = el.split(':');
```

Comment:
```suggestion
    const [key, value] = el.trim().split(':'); // Instead of the map method, since you already have all the elements in that place
```